### DISCUSSION TITLE
min/max/average for a forecast

### DISCUSSION THREAD

**Discussion by gregory1g** [camaraproject/PopulationDensityData#22] [General]
maximum, minimum and average have clear definitions when historical data is exposed.

When one provides future prediction, "average" does not have clear definition. Usually one predicts a value as such. Like we predict that 8000 people participate this event.

maximum and minimum do not have a standard definitions for a prediction either. For a prediction people usually talk about precision or accuracy represented via standard deviation, where an interval +- 2sigma covers majority of probable values.
![https://en.wikipedia.org/wiki/Standard_deviation#/media/File:Standard_deviation_diagram.svg](https://en.wikipedia.org/wiki/Standard_deviation#/media/File:Standard_deviation_diagram.svg)

But to deliver a standard deviation one must:
1) be sure that result has a normal distribution. This can be not the case for population, for example one forecast 8000 people if the weather is ok and down to 100 if it bad. This is not a normal distribution.
2) one must be able to calculate the standard deviation 

Therefore, I would adjust the API response to just return an expected (forecasted) population density value without additional values.

All that imho.

**Comment by jgarciahospital** [comment]
As in #23 , discuss how to provide a definition matching the mathematical calculation.
